external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -1521,12 +1545,10 @@ class AvroV2Suite extends AvroSuite {
         .write
         .format("avro")
         .partitionBy("p1", "p2")
-        .option("header", true)
```

Comment:
Ur, BTW, why do you piggy-back this removal into this PR?

Reply:
The change is small, since I am here I remove unneeded option. Do you want to see a separate PR for the little change?